src/runtime-rs/crates/hypervisor/src/dragonball/inner_device.rs

```diff
@@ -103,8 +100,14 @@ impl DragonballInner {
 
     pub(crate) async fn update_device(&mut self, device: DeviceType) -> Result<()> {
         info!(sl!(), "dragonball update device {:?}", &device);
-
-        Ok(())
+        match device {
+            DeviceType::ShareFs(sharefs_mount) => {
+                // It's safe to unwrap device config as device_config is always there.
```

Comment:
It seems that you should s/device/mount in this comment to match the code.

Reply:
fixed